### ISSUE TITLE
[Bug] Subfolder asset has incorrect path

### ISSUE THREAD

**Issue by voknelis** [laynezh/vite-plugin-lib-assets#53]
Hello,
I've discovered an issue in the plugin that affects asset paths during the library build.

## Description
In the library I maintain I strictly preserve original file structure 'as is' when building the library, including handling assets. That is why I customize asset paths in plugin via `outputPath` setting.
However, I've noticed that in certain scenarios, the plugin generates asset paths without the './' prefix, even when the asset and its import placed on the same folder level. This prefix './' omission causes most bundlers to interpret the asset import as library import, resulting in an error.

## Current behaviour
The plugin generates asset paths without the './' prefix when referring to assets on the same folder level.

## Expected behaviour
The plugin should consistently prepend the './' prefix to generated asset paths when referencing assets on the same folder level.

## Context (Environment)
- Vite v5 (^5.0.4);
- Vue v3 (^3.3.9);
- @laynezh/vite-plugin-lib-assets v0.5.11

## Steps to Reproduce
1. Open the [reproduction repo sandbox](https://codesandbox.io/p/devbox/vue3-vite-assets-import-issue-pxxcgx);
2. Navigate to the `src/components/logos.ts` and examine the following import line:
    ```javascript
    import logo from "./assets/vue-copy.svg";
    ```
3. Execute `yarn build` command;
4. Navigate to the `dist/components/logos.js` and examine the following import line:
    ```javascript
    import logo from "assets/vue-copy.svg";
    ```

--
Let me know, if you need any additional information.

**Comment by coder-layne** [comment]
@voknelis  It's fixed in v0.5.12, you can view the results online [here](https://codesandbox.io/p/devbox/vue3-vite-assets-import-issue-forked-34fc8q).

**Comment by voknelis** [comment]
@coder-layne Thank you, can confirm it's working now 👍
